dietpi/dietpi-software

```diff
@@ -12015,11 +12015,13 @@ _EOF_
 			# remove the service that isn't needed for an app install on DietPi and install the rest
 			G_EXEC cd /tmp/adsb-feeder/src/modules/adsb-feeder/filesystem/root
 			G_EXEC rm ./usr/lib/systemd/system/adsb-bootstrap.service
+			G_EXEC rm ./usr/lib/systemd/system/adsb-update.service
+			G_EXEC rm ./usr/lib/systemd/system/adsb-update.timer
```

Comment:
Shall we add a [patch](https://github.com/MichaIng/DietPi/blob/dev/.update/patches) to remove this service+timer on next DietPi update, if present? Otherwise it would currently remain on systems which did install ADS-B Feeder with DietPi v8.22 already.

Reply:
yes, that makes sense.